apps/connection/src/connection.repository.ts

```diff
@@ -149,7 +149,7 @@ export class ConnectionRepository {
           break;
       }
 
-      const agentDetails = await this.prisma.connections.upsert({
+      const connectionDetails = await this.prisma.connections.upsert({
```

Comment:
it's good the change the unrelated veriable names

Reply:
Yes